modules/evm/rpc/src/lib.rs

```diff
@@ -108,6 +114,18 @@ where
 
 		let api = self.client.runtime_api();
 
+		let balance_value = if let Some(value) = value {
+			to_u128(value).and_then(|v| TryInto::<Balance>::try_into(v).map_err(|_| ()))
```

Comment:
I don't like this. I know it won't fail as long as Balance is u128 but I don't like this approach